tests/includes/wait-for.sh

```diff
@@ -12,8 +12,8 @@ wait_for() {
     name=${1}
     query=${2}
 
-    # shellcheck disable=SC2046,SC2143
-    until [ $(juju status --format=json 2> /dev/null | jq "${query}" | grep "${name}") ]; do
+    # shellcheck disable=SC2143
+    until [ "$(juju status --format=json 2> /dev/null | jq "${query}" | grep "${name}")" ]; do
```

Comment:
I needed to add these quotes because the query I was passing to `wait_for` returned a value with spaces in it, which caused `[` to freak out. (Took me ages to work out - I'm not super-experienced with debugging shell.)

Reply:
Ah, yes, good shout.